### ISSUE TITLE
Server side ragdolls

### ISSUE THREAD

**Issue by sekwah41** [sekwah41/SekC-Physics#18]
Create code to parse over the nodes and only tag the tracked nodes as needed to be sent to the client then remove the rest for client updates. Add a blue debug render colour to know which is which.

Need to do a handshake to the server to see if the mod is installed.

Look to use a multi part entity for triggering pressure plates or anything.

**Comment by James103** [comment]
Does that help with #25 (since both the ragdolls and the "entity killed by" info should reside on the server once this is implemented)?

**Comment by sekwah41** [comment]
Yep, it would help a lot though I need to look into handle it properly so that death drops and events are handled properly for custom mobs.

I would just need to add a hook or something to pass the knockback hit value over to the entity. I believe it would be possible to just simulate the player having hit the entity and then make it handle all the code within the ragdoll itself. Server side ragdolls would also add the ability to add velocity when any player hits the ragdoll easier.

I will be experimenting with adding these as a multi part entity to handle specific locations being hit and also as a few other methods.
